src/main/java/uk/gov/hmcts/probate/controller/DocumentController.java

```diff
@@ -354,7 +354,21 @@ public ResponseEntity<CallbackResponse> reprint(@RequestBody CallbackRequest cal
 
     @PostMapping(path = "/evidenceAdded", consumes = MediaType.APPLICATION_JSON_VALUE)
     public ResponseEntity<CallbackResponse> evidenceAdded(@RequestBody CallbackRequest callbackRequest) {
-        evidenceUploadService.updateLastEvidenceAddedDate(callbackRequest.getCaseDetails());
+        CaseDetails caseDetails = callbackRequest.getCaseDetails();
+        CaseData caseData = caseDetails.getData();
+        if (caseDetails.getState().equalsIgnoreCase("BOCaseStopped")) {
+            log.info("Case is stopped: {} ", caseDetails.getId());
+            if (caseData.getDocumentUploadedAfterCaseStopped() != null
```

Comment:
Did we talk about the addition of this new field documentUploadedAfterCaseStopped, needing a similar update in probate-commons?

Reply:
https://github.com/hmcts/probate-commons/pull/150/files
let me know if its lacking / if more is needed in it